tests/test_node.py

```diff
@@ -1,16 +1,18 @@
+from os import symlink
+
 import pytest
 
 from scantree import CyclicLinkedDir, DirNode, LinkedDir, RecursionPath
 from scantree.test_utils import get_mock_recursion_path
 
 
-def create_basic_entries(local_path):
+def create_basic_entries(local_path: str):
     d1 = local_path.join("d1")
     d1.mkdir()
     f1 = local_path.join("f1")
     f1.write("file1")
-    local_path.join("ld1").mksymlinkto(d1)
```

Comment:
Was this broken somehow (on windows)?

Reply:
(Just curious)

Reply:
yeah, i had to even try it on Win machine, that method is not present for some reason, so I've switched to os independent approach